### PR TITLE
Just call Py_DECREF()

### PR DESCRIPTION
**Pull request by jpereira**

### CODE REVIEW

**Comment by arr2036** [comment]
Looks like you have some scan issues to fix

**Comment by jpereira** [comment]
Argghh!! Now is fixed @arr2036

File: src/modules/rlm_python3/rlm_python3.c [hunk 1]
```diff
@@ -221,46 +219,23 @@ static void python_error_log(void)
...
-		Py_DecRef(pExcTraceback);
-	}
-	if (pStr1) {
-		Py_DecRef(pStr1);
-	}
-	if (pStr2) {
-		Py_DecRef(pStr2);
-	}
+	Py_DECREF(pExcType);
+	Py_DECREF(pExcValue);
```
Comment (arr2036) [inline src/modules/rlm_python3/rlm_python3.c]: What about the rest of the objects?

Comment (arr2036) [reply src/modules/rlm_python3/rlm_python3.c]: Never mind, it's all been moved around

**Pull request merged**